Cargo.toml

```diff
@@ -3,8 +3,8 @@ name = "bitflags"
 # NB: When modifying, also modify:
 #   1. html_root_url in lib.rs
 #   2. number in readme (for breaking changes)
-version = "2.0.0-rc.3"
-edition = "2018"
+version = "2.0.0"
+edition = "2021"
```

Comment:
Should the MSRV be updated if the edition is being bumped to 2021?

Reply:
Ah yes we should do that 👍 I’ll add a CI job for it too since we don’t appear to have one already.

Reply:
Now that the MSRV is 1.56 rust-version can be set in Cargo.toml as well.